MyLibrary/Sources/ScheduleFeature/Schedule.swift

```diff
@@ -104,6 +105,23 @@ public struct Schedule {
         #elseif os(visionOS)
           return .run { _ in await openURL(url) }
         #endif
+      case let .view(.favoriteIconTapped(session)):
+        switch state.selectedDay {
+        case .day1:
+          state.day1 = update(state.day1!, togglingFavoriteOf: session)
```

Comment:
I think saving `Conference` object itself is not good approach. This leads such a whole updating object.
Instead, It might be better for creating `[Days: Favorite]`, which having session hash (I know `session.id` is better but I don't believe `id` in handmade-JSON 😶‍🌫️ ).

Reply:
> Instead, It might be better for creating [Days: Favorite], which having session hash

Thank you.
I've chosen easier way came up with, but will try better approach of performance.

Reply:
To create save method for `[Days: Favorite]`, `Days` must be shown from DataClient module.
However `Days` is a type of `LocalizedStringKey` that is a frozen struct of `SwiftUI`.
Therefore I think it might be better to create save method for `[Session: Bool]`, what do you think?

Further background...
1. If putting `Days` in `SharedModels` and save `[Days: [Sesson]]`
    - `SharedModels` depends on `SwiftUI`, I think `SharedModels` wants to depends only on `Foundation`.
2. If putting `Days` still in `ScheduleFeature` and make it public
    - `DataClient` uses `Days` for determining save method, and of course `ScheduleFeature` depends on `DataClient` already, so there's circular dependence.

Therefore, I resulted in it's better to create `[Session: Bool]` as `Favorites` in `SharedModels`, what do you think of this design idea?

Reply:
Sorry, I realized that it's better to use `Conference` as key instead of enum `Days`!

Reply:
@d-date In 846098f979eb18c7dff4be8e3d7261139302f177 , changed save data type to `[String: [Session]]` which key is enforced to get from `Conference.title`.